docs/data/date-pickers/validation/validation.md

```diff
@@ -108,6 +108,10 @@ If you know that all days of some months are disabled—you can provide the [`sh
 Same with the [`shouldDisableYear`](#disable-specific-years) prop for the `year` view.
 :::
 
+:::info
```

Comment:
I agree with @LukasTy, it should probably be a success (see https://www.notion.so/mui-org/Callouts-and-their-usage-in-the-docs-7e709a02c72142cd8b1a0829861435c5)

Reply:
Honestly, all the variants, except `error` work here. 😆 
However, I'd probably avoid having two warning callouts in a row, thus we are left with either `info` or `success`. 👍

Reply:
OK, fine for me :+1:

Reply:
changed it to `success` since conveying positively is better IMO